src/pages/quickstarts/native/expo.mdx

```diff
@@ -33,21 +33,42 @@ Once you have a Expo application ready, you need to install Clerk's Expo SDK.
   </Tab>
 </Tabs>
 
-## Mount `<ClerkProvider>`
+<Callout type="warning">
+  Ensure you also setup [Expo AuthSession](https://docs.expo.dev/versions/latest/sdk/auth-session/) installed before continuing, as it is a required dependency.
+</Callout>
+
+## Set Environment Variables
 
-Clerk requires your Expo application to be wrapped in the `<ClerkProvider/>` component. The `<ClerkProvider />` component provides active session and user context to Clerk's hooks and other components.
+Below is an example of an `.env.local` and `app.config.js` files. To get your keys go to the API Keys page on the [Clerk dashboard](https://dashboard.clerk.com).
 
-The `CLERK_PUBLISHABLE_KEY` can be obtained from the API Keys page in the [Clerk dashboard](https://dashboard.clerk.dev).
+```sh copy filename=".env.local"
+CLERK_PUBLISHABLE_KEY=pk_test_••••••••••••••••••••••••••••••••••
+```
+
+```sh copy filename="app.config.js"
```

Comment:
Is it worth to add `eas.json` example as well? Or just redirect to [Environment variables and secrets](https://docs.expo.dev/build-reference/variables/)

Reply:
Updated to
![image](https://github.com/clerkinc/clerk-docs/assets/28108272/429cbd82-a7ae-4678-8ef2-4c0e778a3594)